builds/mariner/SPECS/aziot-edge/aziot-edge.spec

```diff
@@ -0,0 +1,193 @@
+%define iotedge_user iotedge
+%define iotedge_group %{iotedge_user}
+%define iotedge_home %{_localstatedir}/lib/aziot/edged
+%define iotedge_logdir %{_localstatedir}/log/aziot/edged
+%define iotedge_socketdir %{_localstatedir}/lib/iotedge
+%define aziot_confdir %{_sysconfdir}/aziot
+%define iotedge_confdir %{aziot_confdir}/edged
+
+Name:           aziot-edge
+Version:        @@VERSION@@
+Release:        @@RELEASE@@%{?dist}
```

Comment:
https://github.com/Azure/iotedge/blob/release/1.2/edgelet/Makefile#L137-L140
I think we use `@version@` and `@release@` as placeholders

Reply:
Yep these are also being uses as placeholders here. they are populated in package-mariner.sh